modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -172,71 +172,71 @@ The Popup component automatically handles right-to-left rendering.
 
 ### Popper
 <>
-<SymbolDoc name="Popper" />
-<SymbolDoc name="Popup" hideDescription />
+<SymbolDoc name="Popper" fileName="/react/" />
```

Comment:
I thought these defaulted to `react`?

Reply:
There is no defaults when it comes to `fileName`. I'm leaving nothing up to chance and instead hardcoding a `fileName` to the package name folder. This way if we add a labs Popup later, we won't accidentally start picking up the labs version. The `find` logic looks for the first match and there is no order guarantee when parsing.